app/src/main/java/com/zulip/android/activities/LoginActivity.java

```diff
@@ -15,6 +15,7 @@
 import android.support.v7.widget.Toolbar;
 import android.text.Editable;
 import android.text.InputType;
+import android.text.StaticLayout;
```

Comment:
It was added by mistake while I was testing something! My bad! I'll remove it! :+1: